packages/fuels-accounts/src/predicate.rs

```diff
@@ -40,10 +40,17 @@ impl Predicate {
     }
 
     pub fn load_from(file_path: &str) -> Result<Self> {
-        let code = fs::read(file_path).map_err(|e| {
+        let absolute_file_path = path::absolute(file_path).map_err(|e| {
             error!(
                 IO,
-                "could not read predicate binary {file_path:?}. Reason: {e}"
+                "failed to make path absolute: {file_path:?}. Reason: {e}"
+            )
+        })?;
+        let code = fs::read(&absolute_file_path).map_err(|e| {
+            let clean_file_path = path_clean::clean(absolute_file_path);
+            error!(
+                IO,
+                "could not read predicate binary {clean_file_path:?}. Reason: {e}",
```

Comment:
Can we read only the provided `file_path` and leave any prettifying (such as getting the abs path) in the Err case?